app/uk/gov/hmrc/nisp/services/NationalInsuranceService.scala

```diff
@@ -59,9 +59,9 @@ class NationalInsuranceService @Inject()(nationalInsuranceConnector: NationalIns
           Left(Exclusion.IsleOfMan)
         case Upstream4xxResponse(message, ExclusionErrorCode, _, _) if message.contains(ExclusionCodeMarriedWomen) =>
           Left(Exclusion.MarriedWomenReducedRateElection)
-        case WithStatusCode(FORBIDDEN, ex) if ex.message.contains(ExclusionCodeCopeProcessingFailed) =>
```

Comment:
Why was `WithStatusCode(FORBIDDEN, ex)` removed? This functionality has now changed not sure if thats what you want

Reply:
WithStatusCode now only takes an UpstreamErrorResponse and returns an option of Int instead of an option of (Int, UpstreamErrorResponse) meaning that we couldn't access the exception for the if block

Reply:
You can preface any match within a case with `somevar@` where `somevar` is a variable that you can access. So you could write this like `ex@WithStatusCode(FORBIDDEN) if ex.message.contains(ExclusionCodeCopeProcessingFailed) => `